docs/changelog.rst

```diff
@@ -30,6 +30,9 @@ New features:
   :bug:`5829`
 * :doc:`plugins/mbcollection`: When getting the user collections, only consider
   collections of releases, and ignore collections of other entity types.
+* :doc:`plugins/mpdstats`: Add new configuration option,
+  ``played_ratio_threshold``, to allow configuring the percentage the song must
```

Comment:
Make sure that the option is called the same way in the code and the docs - currently it is `ratio_played_threshold`